multiqc/utils/report.py

```diff
@@ -32,37 +32,67 @@
 except NameError:
     pass  # Python 3
 
+
 # Set up global variables shared across modules
-general_stats_data = list()
-general_stats_headers = list()
-general_stats_html = ""
-data_sources = defaultdict(lambda: defaultdict(lambda: defaultdict()))
-plot_data = dict()
-html_ids = list()
-lint_errors = list()
-num_hc_plots = 0
-num_mpl_plots = 0
-saved_raw_data = dict()
-last_found_file = None
-runtimes = {
-    "total": 0,
-    "total_sp": 0,
-    "total_mods": 0,
-    "total_compression": 0,
-    "sp": defaultdict(),
-    "mods": defaultdict(),
-}
-file_search_stats = {
-    "skipped_symlinks": 0,
-    "skipped_not_a_file": 0,
-    "skipped_ignore_pattern": 0,
-    "skipped_filesize_limit": 0,
-    "skipped_no_match": 0,
-}
-
-# Make a dict of discovered files for each seach key
-searchfiles = list()
-files = dict()
+def init():
```

Comment:
```suggestion
# Inside a function so that the global vars are reset if MultiQC is run more than once within a single session / environment
def init():
```